.travis.yml

```diff
@@ -1,4 +1,3 @@
 language: node_js
 node_js:
-  - "iojs"
-  - "6"
+  - "6"
```

Comment:
no carriage return here. Shouldn't matter but github shows us w/ the red do-not-enter symbol